src/ResizeScreen.js

```diff
@@ -14,7 +14,7 @@ export default class ResizeScreen extends React.Component {
 
   canvasRef = React.createRef();
   firstFrame = null;
-  // We stare these as instance properties rather than on state because we
+  // We start these as instance properties rather than on state because we
```

Comment:
```suggestion
  // We store these as instance properties rather than on state because we
```

Reply:
Yeah nah that makes wayyyy more sense. Will add a TODO to fix

Reply:
I legitimately didn't even check the context. 

Merged.